config/settings.d/externalipam.yml.example

```diff
@@ -0,0 +1,8 @@
+---
+:enabled: false
+
+# Supported providers:
```

Comment:
Technically any gem can add supported providers. Perhaps we should rephrase this to
```suggestion
# Built-in providers:
```

Reply:
I like this wording, "support" creates so much confusion.